Gruntfile.js

```diff
@@ -86,16 +117,19 @@ module.exports = function(grunt) {
     grunt.loadNpmTasks('grunt-contrib-watch');
     grunt.loadNpmTasks('grunt-contrib-connect');
     grunt.loadNpmTasks('grunt-css');
+    grunt.loadNpmTasks('grunt-eslint');
+    grunt.loadNpmTasks('grunt-open');
     grunt.loadNpmTasks('grunt-shell');
     grunt.loadNpmTasks('grunt-zip');
     grunt.loadNpmTasks('grunt-s3');
     grunt.loadNpmTasks('grunt-clean');
     grunt.loadNpmTasks('grunt-release');
 
     // Default task(s).
+    grunt.registerTask('examples', ['build', 'connect:server', 'open:examples', 'watch']);
+    grunt.registerTask('lint', ['eslint:prod']);
     grunt.registerTask('serve', ['connect', 'watch']);
-    grunt.registerTask('build', ['uglify', 'cssmin', 'zip']);
-    grunt.registerTask('publish', ['build', 'release', 's3'])
-    grunt.registerTask('default', 'build')
-
-};
\ No newline at end of file
+    grunt.registerTask('build', ['lint', 'uglify', 'cssmin', 'zip']);
+    grunt.registerTask('publish', ['build', 'release', 's3']);
```

Comment:
I haven't looked at all what this task does / if it's even necessary. I'll look into it further and update accordingly.